python/metashape_workflow.py

```diff
@@ -14,69 +14,68 @@
 # ---- If not running interactively, the config file should be supplied as the command-line argument after the python script, e.g.: python metashape_workflow.py config.yml
 
 
-## Load custom modules and config file: slightly different depending whether running interactively or via command line
+# Load custom modules and config file: slightly different depending whether running interactively or via command line
 try:  # running interactively (in linux) or command line (windows)
-    from python import metashape_workflow_functions as meta
-    from python import read_yaml
+    from python.metashape_workflow_functions import MetashapeWorkflow
 except:  # running from command line (in linux) or interactively (windows)
-    import metashape_workflow_functions as meta
-    import read_yaml
+    from metashape_workflow_functions import MetashapeWorkflow
```

Comment:
If we set this up as a python module, we shouldn't have to worry about having two options for how to import it.